src/simulator_core/entities/assets/pipe.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, asset_name: str, asset_id: str):
 
         :param str asset_name: The name of the asset.
         :param str asset_id: The unique identifier of the asset.
-        :param PandapipesNet pandapipe_net: Pandapipes network object to register asset to.
+        :param any geometry: ESDL geometry
```

Comment:
Can you remove this line? Probably a left over from the previous implementation.